cmd/main.go

```diff
@@ -115,6 +117,8 @@ func main() {
 	flag.IntVar(&webhookPort, "webhook-port", 9443, "Admission webhook port.")
 	flag.StringVar(&webhookCertDir, "webhook-cert-dir", "/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.")
+	flag.IntVar(&pprofPort, "pprof-port", 0, "Port that the controller should bind to for serving pprof, 0 or empty value disables pprof")
```

Comment:
Would be nicer to have similar to `metricsAddr` semantics so that optionally pprof endpoint could be on `0.0.0.0` and not strictly on localhost

Reply:
This is OSS code, so like nothing really to hide